controllers/administration.js

```diff
@@ -1168,8 +1168,7 @@ router.all('/students', permissionsHelper.permissionsChecker(['ADMIN_VIEW', 'STU
 
 const getUsersWithoutConsent = async (req, roleName, classId) => {
     const role = await api(req).get('/roles', { qs: { name: roleName }, $limit: false });
-    const qs = { roles: role.data[0]._id };
-
+    const qs = { roles: role.data[0]._id, $limit: false };
```

Comment:
Show line 1186: " users = (await api(req).get('/users', { qs, $limit: false })).data;" 
$limit is set in different levels.